applications/ParticleMechanicsApplication/particle_mechanics_application_variables.cpp

```diff
@@ -75,6 +77,7 @@ namespace Kratos
     KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( MPC_COORD )
     KRATOS_CREATE_VARIABLE( int, MPC_CONDITION_ID )
     KRATOS_CREATE_VARIABLE( bool, MPC_IS_NEUMANN )
+    KRATOS_CREATE_VARIABLE( int, MPC_BOUNDARY_CONDITION_TYPE )
```

Comment:
i would use an enum to better identify

Reply:
@philbucher here, how would you define an enum as a variable?
Is this done somewhere?

Basically @VeronikaSinger needs an identifier for each condition. A `Flag` is not enough, since it can be multiple things, not just penalty or lagrange

What would you suggest?